float32 sample_time
float32 update_time
p3dx_motionplanner/P3DXPose[] fleet_config
p3dx_motionplanner/P3DXPose[] init_pose
p3dx_motionplanner/P3DXPose[] terminal_pose
Room room
Obstacle[] obstacles
int32[] robobst
